dbt_subprojects/dex/models/trades/test/dex_ci_trades_test.sql

```diff
@@ -0,0 +1,53 @@
+{{ config(
+    schema = 'dex'
+    , alias = 'ci_trades_test'
+    , materialized = 'view'
+    , tags = ['prod_exclude']
+    )
+}}
+
+-- This view is only for CI testing purpose
+-- It directly processes modified base_trades models and enriches them with token information
+
+{% set modified_models = get_modified_base_trades() %}
+{% set git_schema = 'git_dunesql_' ~ env_var('GIT_SHA', '') %}
+
+WITH base_trades AS (
+    {% for file in modified_models %}
+    {% set model_name = file.split('/')[-1].replace('.sql', '') %}
+    SELECT
+        blockchain,
+        project,
+        version,
+        block_month,
+        block_date,
+        block_time,
+        block_number,
+        token_bought_amount_raw,
+        token_sold_amount_raw,
+        CAST(token_bought_address AS varbinary) as token_bought_address,
+        CAST(token_sold_address AS varbinary) as token_sold_address,
+        CAST(taker AS varbinary) as taker,
+        CAST(maker AS varbinary) as maker,
+        CAST(project_contract_address AS varbinary) as project_contract_address,
+        tx_hash,
+        evt_index,
+        cast(null as varbinary) as tx_from,  -- For testing only, not used in production
+        cast(null as varbinary) as tx_to     -- For testing only, not used in production
+    FROM delta_prod.test_schema.{{ git_schema }}_{{ model_name }}
+    WHERE block_date = current_date - interval '1' day
```

Comment:
should we expand the sample size to be a bit larger, as the large amounts can be "hidden" in the history? maybe test a week, a month, see how it looks

Reply:
```suggestion
    WHERE block_date = current_date - interval '30' day
```